clouddriver-lambda/src/main/java/com/netflix/spinnaker/clouddriver/lambda/deploy/ops/AbstractLambdaAtomicOperation.java

```diff
@@ -16,24 +16,15 @@
 
 package com.netflix.spinnaker.clouddriver.lambda.deploy.ops;
 
-import com.amazonaws.services.lambda.AWSLambda;
-import com.netflix.spinnaker.clouddriver.aws.security.AmazonClientProvider;
-import com.netflix.spinnaker.clouddriver.aws.security.AmazonCredentials;
-import com.netflix.spinnaker.clouddriver.aws.security.NetflixAmazonCredentials;
 import com.netflix.spinnaker.clouddriver.data.task.Task;
 import com.netflix.spinnaker.clouddriver.data.task.TaskRepository;
 import com.netflix.spinnaker.clouddriver.lambda.deploy.description.AbstractLambdaFunctionDescription;
-import com.netflix.spinnaker.clouddriver.lambda.deploy.exception.InvalidAccountException;
 import com.netflix.spinnaker.clouddriver.lambda.provider.view.LambdaFunctionProvider;
 import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperation;
-import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider;
 import org.springframework.beans.factory.annotation.Autowired;
 
 public abstract class AbstractLambdaAtomicOperation<T extends AbstractLambdaFunctionDescription, K>
-    implements AtomicOperation<K> {
-  @Autowired AmazonClientProvider amazonClientProvider;
-
-  @Autowired AccountCredentialsProvider accountCredentialsProvider;
+    extends AbstractLambdaProvider implements AtomicOperation<K> {
```

Comment:
Would it make sense to do composition instead of inheritance here?

Reply:
I'd LEAN towards... at this point.. leaving it where it is.  Renamed the class, but all of this could use a more significant refactor in the future.